kokoro/linux/python/x86_64/kokoro_build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -eux
+#
+# This is the top-level script we give to Kokoro as the entry point for
+# running Python tests. The tests themselves run in a Docker container,
+# specified by test_container.sh.
+
+# The following values can be overridden by environment variable:
+#   DOCKER_IMAGE
+#   PROTOBUF_WORKSPACE
+#   PROTOBUF_PYTHON_RUNTIME (used in build_python.sh)
+
+# The Docker image to run the build.
+#
+# The official Python images are based on buildpack-deps, which is
+# sufficient to build the Protobuf project. The versioned builds specify
+# the image via environment variable, but this is a reasonable default:
+: ${DOCKER_IMAGE:=python:buster}
+
+# The location for the source checkout.
```

Comment:
Can this support interactive invocation from a working directory, for easy testing during development? I recently did this for our PHP images:

https://github.com/protocolbuffers/protobuf/blob/master/kokoro/linux/php_all/build.sh